underfs/tos/src/main/java/alluxio/underfs/tos/TOSOutputStream.java

```diff
@@ -167,7 +167,7 @@ public void close() throws IOException {
       return;
     }
     mLocalOutputStream.close();
-    try (BufferedInputStream in = new BufferedInputStream(new FileInputStream(mFile))) {
+    try (BufferedInputStream in = new BufferedInputStream(Files.newInputStream(mFile.toPath()))) {
```

Comment:
Why did you change the implementation here?

Reply:
Because this way is recommended in effective java.